src/win32/wazuh-installer.wxs

```diff
@@ -71,6 +71,7 @@
         <Property Id="ApplicationFolderName" Value="ossec-agent" />
         <CustomAction Id="LaunchApplication" BinaryKey="WixCA" DllEntry="WixShellExec" Impersonate="no" />
         <Binary Id="InstallerScripts" SourceFile="InstallerScripts.vbs" />
+        <Binary Id="CMDPath" SourceFile="C:\Windows\System32\cmd.exe" />

```

Comment:
As per conversations, the path can't be harcoded. @okynos will develop another approach.